Soccar [Game-Server]/src/nl/soccar/gameserver/model/session/GameWrapper.java

```diff
@@ -82,6 +79,7 @@ public void stop() {
     public void sendWorldObjects(PlayerWrapper player) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)